addon-test-support/@ember/test-helpers/dom/click.js

```diff
@@ -9,16 +9,17 @@ import isFormControl from './-is-form-control';
 /**
   @private
   @param {Element} element the element to click on
+  @param {Object} options the options to be merged into the mouse events
 */
-export function __click__(element) {
-  fireEvent(element, 'mousedown');
+export function __click__(element, options) {
+  fireEvent(element, 'mousedown', options);
 
   if (isFocusable(element)) {
     __focus__(element);
   }
 
-  fireEvent(element, 'mouseup');
-  fireEvent(element, 'click');
+  fireEvent(element, 'mouseup', options);
```

Comment:
It seems really odd to me to have the same options for mousedown, mouseup, and click 🤔

Reply:
The common options are going to be button, x, y. Which options would you want to be different for each event?